src/Text/Libyaml.hs

```diff
@@ -520,8 +553,13 @@ newtype ToEventRawException = ToEventRawException CInt
 instance Exception ToEventRawException
 
 decode :: MonadResource m => B.ByteString -> ConduitM i Event m ()
-decode bs | B8.null bs = return ()
-decode bs =
+decode = mapOutput yamlEvent . decodeMarked
+
+-- |
+-- @since [INSERTVERSION]
```

Comment:
The version will be 0.10.4.0, please add it here, update the cabal file, and make an entry in the ChangeLog.